src/transformers/modeling_longformer.py

```diff
@@ -96,7 +90,7 @@ class LongformerBaseModelOutput(ModelOutput):
             be accessed from 'global_attentions'.
         global_attentions (:obj:`tuple(torch.FloatTensor)`, `optional`, returned when ``output_attentions=True`` is passed or when ``config.output_attentions=True``):
             Tuple of :obj:`torch.FloatTensor` (one for each layer) of shape
-            :obj:`(batch_size, num_heads, x, sequence_length)`, where `x` is the number of tokens with global
+            :obj:`(batch_size, num_heads, sequence_length, x)`, where `x` is the number of tokens with global
```

Comment:
I changed the ordering here - I think it's better to be consistent with `attentions` in that first dim is the whole `seq_len` and the second dimension is `x` or `x + attn_window + 1`.